### PR TITLE
feat(Accordion): Added Accordion component

### PR DESCRIPTION
**Pull request by woutsluiter**
### This PR:

**Breaking changes** 🔥
- none

**Backwards compatible additions** ✨
- `Accordion` component

**Bugfixes/Changed internals** 🎈
- none

**Checklist** 🛡
- [x] I have exported my addition from `src/index.ts` (check if not applicable).
- [x] Appropriate tests have been added for my functionality (check if not applicable).
- [x] A designer has seen and approved my changes (tag `@LuukHorsmans` or `@RianneSchaekens` for a design review when applicable).
- [x] I have tested my addition in all supported browsers and for responsiveness (Chrome, Firefox, Safari, Edge, IE11 and mobile browsers).
- [x] Appropriate documentation has been written (check if not applicable).

### CODE REVIEW

File: packages/components/src/components/Accordion/test.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Accordion from '.';
+import { mountWithTheme } from '../../utility/styled/testing';
+import Text from '../Text';
+import FoldOut from '../FoldOut';
+
+describe('Accordion', () => {
+    it('should render a label and content a foldout icon', () => {
```
Comment (hborghols) [inline packages/components/src/components/Accordion/test.tsx:8]: This is not the sentence you intended I think 😛

Comment (woutsluiter) [reply packages/components/src/components/Accordion/test.tsx:8]: Find you this a not good sentence??

Comment (hborghols) [reply packages/components/src/components/Accordion/test.tsx:8]: good I find it not young wood

Comment (woutsluiter) [reply packages/components/src/components/Accordion/test.tsx]: Now I fix it then well

**Pull request merged**